### DISCUSSION TITLE
Display Colours Popout EWD SD

### DISCUSSION THREAD

**Discussion by MaexliBaexli** [flybywiresim/aircraft#7799] [Help]
Good day,

i always used to fly with the Stabe version of FBW. SInce yesterday i changed to the Developer Version because of the stutter fix. As i always pop out my displays i now have different display colours (only on one monitor) on my Engine Warning Display and Status Display. 

Is there any difference concerning displays on both versions? 

Thanks again for your work!

Greets,
Markus

![8c43921cbdafc5da00fe98643deda0034e5ac768](https://user-images.githubusercontent.com/114825031/218954137-77d7260e-b7e8-4d4a-b415-55c647a5f831.jpeg)

**Comment by tracernz** [comment]
I can see the same. Please raise a bug report https://github.com/flybywiresim/a32nx/issues/new/choose

**Reply by MaexliBaexli** [reply]
Thanks for your fast reply, will make the bug report later on,

Greets,
Markus

**Reply by tracernz** [reply]
Scratch the bug report. I've made a fix https://github.com/flybywiresim/a32nx/pull/7800

**Reply by MaexliBaexli** [reply]
No way, great. Is the fix already applied? I am on DEV Version,

Greets,
Markus

**Reply by MaexliBaexli** [reply]
> Scratch the bug report. I've made a fix #7800

WORKING, GREAT, Thanks a lot,

Greets,
Markus

![Screenshot_20230215_111450_com android gallery3d](https://user-images.githubusercontent.com/114825031/218999824-875a511f-a406-419e-923e-5946f013e492.jpg)
